### PR TITLE
UIIN-1858 Display acquisition accordion on Item record

### PR DESCRIPTION
**Pull request by NikitaSedyx**
## Purpose

https://issues.folio.org/browse/UIIN-1858

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [x] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: translations/ui-inventory/en.json [hunk 1]
```diff
@@ -705,6 +705,9 @@
   "acq.orderType.Ongoing": "Ongoing",
   "acq.dateOrdered": "Date ordered",
   "acq.acqUnit": "Acquisition unit",
+  "acq.orderSubscription": "Subscription",
+  "acq.receivedDate": "Piece receipt date",
```
Comment (AndreiShumski2) [inline translations/ui-inventory/en.json:709]: why receiptDate  === receiptDate ? Piece has receiptDate and receivedDate

**Review by AndreiShumski2 — APPROVED** [review]

**Review by mkuklis — APPROVED** [review]

**Review by usavkov-epam — APPROVED** [review]

File: src/Item/ViewItem/ItemAcquisition/ItemAcquisition.js [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+          />
+        </Col>
+
+        <Col xs={4}>
+          <KeyValue
+            label={<FormattedMessage id="ui-inventory.acq.receiptStatus" />}
+            value={
+              orderLine
+                ? <FormattedMessage id={`ui-inventory.acq.receiptStatus.${orderLine.receiptStatus}`} />
+                : <NoValue />
```
Comment (AndreiShumski2) [inline src/Item/ViewItem/ItemAcquisition/ItemAcquisition.js:82]: as I remember, `KeyValue` returns `NoValue` if the value is not passed or undefined.

Comment (NikitaSedyx) [reply src/Item/ViewItem/ItemAcquisition/ItemAcquisition.js:82]: formatted message is always defined

Comment (NikitaSedyx) [reply src/Item/ViewItem/ItemAcquisition/ItemAcquisition.js:82]: it's pretty the same with &&, but seems saves 1 line

File: src/Item/ViewItem/ItemAcquisition/ItemAcquisition.js [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+            }
+          />
+        </Col>
+
+        <Col xs={4}>
+          <KeyValue
+            label={<FormattedMessage id="ui-inventory.acq.receivedDate" />}
+            value={
+              piece.receivedDate
```
[the remark this answers is not in the document]
Comment (NikitaSedyx) [reply src/Item/ViewItem/ItemAcquisition/ItemAcquisition.js:91]: yep, right

**Pull request merged**